rslib/src/notetype/schema11.rs

```diff
@@ -211,6 +214,9 @@ pub struct NoteFieldSchema11 {
     #[serde(default, deserialize_with = "default_on_invalid")]
     pub(crate) description: String,
 
+    #[serde(default, deserialize_with = "default_on_invalid")]
+    pub(crate) input: String,
```

Comment:
This will make it default to "", which will confuse the front-end code. Instead of using a string here which is not very space efficient, how about either a boolean, or if you anticipate there to be other kinds of editors in the future, an enum?

Reply:
I initially thought the string would be easier for others who see the field data for the first time, but actually a boolean is just as obvious, as long as the name is telling.